subprojects/native/src/main/java/org/gradle/internal/nativeintegration/filesystem/services/NativePlatformBackedFileMetadataAccessor.java

```diff
@@ -35,20 +36,28 @@ public NativePlatformBackedFileMetadataAccessor(Files files) {
     @Override
     public FileMetadataSnapshot stat(File f) {
         try {
-            FileInfo stat = files.stat(f, true);
+            FileInfo stat = files.stat(f, false);
```

Comment:
Only this should be wrapped with a `try`, not the rest of the method.

Reply:
Both calls should be wrapped in a `try ... catch` for the same reason both calls need to in a `try ... catch` for the NIO accessor: We treat errors to stat a file as missing files.

Reply:
Yes. What I'm saying is we shouldn't wrap the whole method in a `try`, only this line.

Reply:
Done.